lang/fr.json

```diff
@@ -0,0 +1,28 @@
+{
+  "VISION5E.Blindsense": "Perception aveugle",
+  "VISION5E.BlindSenses": "Sens aveugles",
+  "VISION5E.Burrowing": "Fouissage",
```

Comment:
The "Flying" status is translated as "En vol" in the French core translation. Does "Fouissage" fit? What about"Fouissant"? I also found "Fouissement" be used for the burrow speed in monster stat blocks. No idea what the difference between "Fouissage" and "Fouissement" is.

Reply:
"Fouissage" is the translation for dnd5e borrowing movement speed in both NPC and PC. 
![image](https://github.com/dev7355608/vision-5e/assets/5630069/3e0f2c2e-9b8a-4fad-8a5d-76a3eceaf3e1)
Fouissage and Fouissement are synonym

Reply:
I think "Enfoui" is more accurate for the effect. That's the state of being underground.